systemtest/src/main/java/io/strimzi/systemtest/utils/specific/MetricsUtils.java

```diff
@@ -53,6 +56,25 @@ public static String getExporterRunScript(String podName, String namespace) thro
         return exec.out();
     }
 
+    public static MetricsCollector setupCOMetricsCollectorInNamespace(String coName, String coNamespace, String coScraperName) {
+
+        String originalCODeploymentName = ResourceManager.getCoDeploymentName();
+        // necessary for finding correct deployment when building metric collector.
+        ResourceManager.setCoDeploymentName(coName);
+
+        String coScraperPodName = ResourceManager.kubeClient().listPodsByPrefixInName(coNamespace, coScraperName).get(0).getMetadata().getName();
+
+        MetricsCollector metricsCollector =  new MetricsCollector.Builder()
+            .withScraperPodName(coScraperPodName)
+            .withNamespaceName(coNamespace)
+            .withComponentType(ComponentType.ClusterOperator)
+            .withComponentName("")
```

Comment:
Maybe not part of this PR, but in case we pass empty string, shouldn't it be a default value?